.github/workflows/sdk-suggestion.yaml

```diff
@@ -0,0 +1,121 @@
+name: Speakeasy SDK Suggestion Workflow
+
+on:
+  workflow_call:
+    inputs:
+      speakeasy_version:
+        description: The version of the Speakeasy CLI to use or "latest"
+        default: latest
+        required: false
+        type: string
+      openapi_doc_auth_header:
+        description: |-
+          The auth header to use when fetching the OpenAPI document if it is not publicly hosted. For example `Authorization`.
+          If using a private speakeasy hosted document use `x-api-key`. This header will be populated with the openapi_doc_auth_token provided.
+        required: false
+        type: string
+      openapi_docs:
+        description: |-
+          A yaml string containing a list of OpenAPI documents to use, if multiple documents are provided they will be merged together, prior to suggestion.
+
+          If the document lives within the repo a relative path can be provided, if the document is hosted publicly a URL can be provided.
+
+          If the documents are hosted privately a URL can be provided along with the `openapi_doc_auth_header` and `openapi_doc_auth_token` inputs.
+          Each document will be fetched using the provided auth header and token, so they need to be valid for all documents.
+
+          For example:
+          openapi_docs: |
+            - https://example.com/openapi1.json
+            - https://example.com/openapi2.json
+        required: false
+        type: string
+      openapi_doc_output:
+        description: "The path to output the modified OpenAPI spec"
+        required: false
+        default: "./openapi.yaml"
+        type: string
+      max_suggestions:
+        description: "The maximum number of suggestions to apply"
+        required: false
+        default: "5"
+        type: string
+    secrets:
+      github_access_token:
+        description: A GitHub access token with write access to the repo
+        required: true
+      openapi_doc_auth_token:
+        description: The auth token to use when fetching the OpenAPI document if it is not publicly hosted. For example `Bearer <token>` or `<token>`.
+        required: false
+      slack_webhook_url:
+        description: A Slack webhook URL that pipeline failures will be posted to
+        required: false
+      speakeasy_api_key:
+        description: The API key to use to authenticate the Speakeasy CLI
+        required: true
+      openai_api_key:
+        description: "The OpenAI API key to authenticate to access LLM suggestions"
+        required: false
+jobs:
+  suggest:
+    name: Suggest OpenAPI Document
+    runs-on: ubuntu-latest
+    outputs:
+      branch_name: ${{ steps.suggest.outputs.branch_name }}
+      cli_output: ${{ steps.suggest.outputs.cli_output }}
+    steps:
+      - name: Tune GitHub-hosted runner network
```

Comment:
What is this for???

Reply:
apparently it improves networking performance on github runners: https://github.com/smorimoto/tune-github-hosted-runner-network We use it in our other workflows as well